apps/coscrad-frontend/src/components/coscrad-logo/coscrad-logo.tsx

```diff
@@ -1,11 +1,11 @@
-// We're importing this asset so that if it is not present it will break the build
-import coscradLogo from '../../assets/coscrad-logo.png';
-import './coscrad-logo.css';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { StyledImageInUILayout } from '../../utils/generic-components/presenters/styled-image-in-ui-layout';
 
 export const COSCRADLogo = (): JSX.Element => {
+    const { coscradLogoUrl } = useContext(ConfigurableContentContext);
+
     return (
-        <span className="coscrad-logo">
-            <img src={coscradLogo} alt={'COSCRAD Logo'} />
-        </span>
+        <StyledImageInUILayout sx={{ width: '100px' }} src={coscradLogoUrl} alt="COSCRAD Logo" />
```

Comment:
The name of this component is a bit funky.

Reply:
Yes, I'm open to suggestions.  I have a story to re-write the image-in-content component so I could implement a new name then if we can come up with one.